docs/user-guide/install-zowe-zos-convenience-build.md

```diff
@@ -80,7 +84,7 @@ After you download the PAX file, you can transfer it to z/OS and expand its cont
    
    This is the Zowe runtime directory and is referred to as `<RUNTIME_DIR>` throughout this documentation.  
 
-   **Note:** Zowe version 1 had a script `zowe-install.sh` that created a separate Zowe runtime directory from the expanded contents of the Zowe PAX file. Zowe v2 no longer has this step. **In Zowe v2, the contents of the expanded Zowe PAX file are the Zowe runtime directory.**
+   **Note:** Zowe version 1 had a script `zowe-install.sh` that created a separate Zowe runtime directory from the expanded contents of the Zowe PAX file. Zowe v2 no longer has this step. **In Zowe v2, and Zowe v3 the contents of the expanded Zowe PAX file are the Zowe runtime directory.**
```

Comment:
Can you please confirm that this note also applies to V3?

Reply:
Good catch!
* Yes, it is valid for Zowe version 2 **and** 3. 
* It would probably make sense to remove the V1 part.